static/usage/v7/select/customization/icon-flip-behavior/vue.md

```diff
@@ -0,0 +1,55 @@
+```html
+<template>
+  <ion-list>
+    <ion-item>
+      <ion-select
+        class="always-flip"
+        toggle-icon="caret-down-sharp"
```

Comment:
Ionic Vue needs the icons imported from `ionicons/icons` and then passed to the props

```typescript
import { caretDownSharp } from 'ionicons/icons';

...

:toggle-icon="caretDownSharp"
```